doc/source/dsintro.rst

```diff
@@ -1047,3 +1047,31 @@ Alternatively, one can convert to an xarray ``DataArray``.
    p.to_xarray()
 
 You can see the full-documentation for the `xarray package <https://xarray.pydata.org/en/stable/>`__.
+
+.. _dsintro.data_type:
+
+Data Types
+----------
+
+Every :class:`Index`, :class:`Series`, and column of a :class:`DataFrame` has a data type.
+The data type or types are available with :meth:`Index.dtype`, :meth:`Series.dtype` and
+:meth:`DataFrame.dtypes` (one dtype per column).
+
+For the most part, pandas uses NumPy arrays and dtypes. Pandas has made a few extensions
+to NumPy for types that are especially important for tabular data analysis.
+
+===================================== ==============================
+Array Type                            Documentation
+===================================== ==============================
+:class:`Categorical`                  :ref:`categorical`
+:class:`DatetimeArray`                TODO
+:class:`api.extensions.IntegerArray`  :ref:`integer_na`
```

Comment:
I think we should be consistent here, moving IntegerArray & PeriodArray into pd namespace